### DISCUSSION TITLE
Development Approach: Getting Started

### DISCUSSION THREAD

**Discussion by chrsnymn** [chrsnymn/redwoodjs-stripe#5] [Announcements]
After making the [Proof of Concept ](https://github.com/redwoodjs/payments)and working on the [Example Store](https://github.com/redwoodjs/example-store-stripe), ie approaching a Redwood Stripe integration from both sides, time has come to actually get down to development of the integration package. Below are the steps I plan to take regarding getting started:

1. Gather Feedback for the Example Store
2. Re-evaluate the Proposal Doc to see what is feasible
3. Create ReadMe (ongoing)
4. Abstracting out low-level chunks of code inside a forked version of the example store
5. Using the low-level abstractions to create high-level abstractions until eventually all the features for MVP are completed
6. Somewhere along the way get more contributors

### MVP Features
- Stripe Checkout for Recurring and One-Time Prices / Products
- Simple webhook integration
- Stripe Customer Portals + Redwoodjs DBAuth integration
   _Possible that it may just be Stripe Customer Portals integration and an integration between Stripe Customers and Redwoodjs DBAuth will be a later addition_

### How to Help
At the moment feedback on the Example Store will be greatly appreciated, specifically with regards to taking advantage of Redwood tools and API suggestions. 

Another way to help out is by contributing to the Example Store. There are currently a few issues open on the Example Store that anyone can take a stab at. 

Once things are a bit more fleshed out over here, the repo will be open to code contributors. However for now, API suggestions based on review of the Proof of Concept and Example Store are most welcome.

That said, any feedback will be appreciated :)

**Comment by adriatic** [comment]
Hello, @chrisvdm I decided that I am very interested in helping you - probably more than just V1 of this project. So, I want to use this discussion tool to keep you updated about my status: Let me know if you do not care to see this (I will keep such diary for my own good anyway)

### 05/302022

Started late, so I did the following:
1. cloned [Example Store Stripe](https://github.com/redwoodjs/example-store-stripe.git)
2. `yarn install` went OK
3. will continue on building, running and checking the [related issues](https://github.com/chrisvdm/redwoodjs-stripe/issues)

**Reply by chrsnymn** [reply]
Hey @adriatic, glad you are interested :)

Thank you for opening up those issues over at the example store. It's a good place to start
